platform/cog-platform-fdo.c

```diff
@@ -1094,6 +1207,16 @@ touch_on_down (void *data,
             &raw_event,
             sizeof (struct wpe_input_touch_event_raw));
 
+    if (popup_data.wl_surface) {
+        if (wl_data.touch.surface == popup_data.wl_surface) {
+            cog_popup_menu_handle_event (popup_data.popup_menu,
+                                         1, raw_event.x, raw_event.y);
```

Comment:
What's does `1` passed to the `state` parameter do here? It would be good to have a symbolic value, like `COG_POPUP_MENU_STATE_<foo>`, or if it is is a boolean flag, use something like:

```c
const bool popup_state_foo = true;
cog_popup_menu_handle_event (…, popup_state_foo, …);
```

Reply:
This means that the 'pressed' event is being emitted.